ui/src/App.tsx

```diff
@@ -20,26 +22,28 @@ const App = () => {
     const [trafficStatsModalOpen, setTrafficStatsModalOpen] = useRecoilState(trafficStatsModalOpenAtom);
 
     return (
-        <StyledEngineProvider injectFirst>
-            <ThemeProvider theme={createTheme(({}))}>
-                <div className="mizuApp">
-                    <Header />
-                    <TrafficPage />
-                    {window["isServiceMapEnabled"] && <ServiceMapModal
-                        isOpen={serviceMapModalOpen}
-                        onOpen={() => setServiceMapModalOpen(true)}
-                        onClose={() => setServiceMapModalOpen(false)}
-                        getServiceMapDataApi={api.serviceMapData} />}
-                    {window["isOasEnabled"] && <OasModal
-                        getOasServices={api.getOasServices}
-                        getOasByService={api.getOasByService}
-                        openModal={oasModalOpen}
-                        handleCloseModal={() => setOasModalOpen(false)}
-                    />}
-                    <TrafficStatsModal isOpen={trafficStatsModalOpen} onClose={() => setTrafficStatsModalOpen(false)} getTrafficStatsDataApi={api.getTrafficStats}/>
-                </div>
-            </ThemeProvider>
-        </StyledEngineProvider>
+        <EuiProvider>
```

Comment:
why its on the whole level of the app?

Reply:
You think its better the wrap only the component that uses the traffic stats from the ui-common?

Reply:
yes, if its not used by the whole app,no need